src/components/Proposals/ProposalPage/OPProposalApprovalPage/OPProposalApprovalPage.tsx

```diff
@@ -63,7 +63,7 @@ export default async function OPProposalApprovalPage({
       alignItems="items-start"
       className={styles.proposal_container}
     >
-      <ProposalDescription proposal={proposal} />
+      <ProposalDescription proposalVotes={proposalVotes} proposal={proposal} />
```

Comment:
My preference would be to make the vote chart a top-level component rather than a child of ProposalDescription - it will be easier to debug this way.

Reply:
@sbvhev gotcha. Sounds good. Let's leave things as is for now.